GVRf/Framework/framework/src/main/jni/objects/scene_object.h

```diff
@@ -180,7 +180,7 @@ class SceneObject: public HybridObject {
     std::vector<SceneObject*> children_;
     float lod_min_range_;
     float lod_max_range_;
-    bool using_lod_;
+    static bool using_lod_;
```

Comment:
Previously, if any object uses lod but the parent is completely inside the frustum the lod check would not be performed because as an optimization the children would not be further evaluated.

I didn't want to overdo this pull request hence the crude solution where if any object uses lod then this global flag disables the aforementioned optimization to ensure the lod check is performed always.

The lod-test has been broken for quite some time (few months) now that I understand what exactly it is doing.